server/apps/rules/routing_rules.py

```diff
@@ -21,6 +22,49 @@
 STATE_ROUTED = 'routed'
 
 
+class Weekdays(Enum):
+    """Weekdays names we use for scheduling."""
+
+    MON = 0
+    TUE = 1
+    WED = 2
+    THU = 3
+    FRI = 4
+    SAT = 5
+    SUN = 6
+
+    @classmethod
+    def is_valid_schedule(cls, list_of_days):
+        """Test if all days in list_of_days are valid day names.
+
+        :param list list_of_days eg. ['mon', 'tue', 'fri']
+        """
+        return all([day.upper() in cls.__members__ for day in list_of_days])
```

Comment:
Maybe just convert the temporary list to a generator expression, but it's very minor.